src/lib/libglemu.js

```diff
@@ -3616,7 +3615,7 @@ var LibraryGLEmulation = {
 
   // OpenGL Immediate Mode matrix routines.
   // Note that in the future we might make these available only in certain modes.
-  glMatrixMode__deps: ['$GL', '$GLImmediateSetup', '$GLEmulation'], // emulation is not strictly needed, this is a workaround
+  glMatrixMode__deps: ['$GL', '$GLImmediateSetup'], // emulation is not strictly needed, this is a workaround
```

Comment:
The comment suggests emulation was needed for some non-direct reason. Do you know what that is?

Reply:
Good point.. perhaps we should leave this in out of an abundance of caution.    I'll see if the git log can tell me why this was needed.